### PR TITLE
docs: fix typo in PR template

### PR DESCRIPTION
**Pull request by snitin315**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Fix typo. `autofixing` -> `auto fixing`

I caught this via Grammarly. 

Screenshot 2021-11-28 at 6 46 55 AM

#### Is there anything you'd like reviewers to focus on?

Is this correct?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 [ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
 [ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
 [ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
-[ ] Add autofixing to a rule
+[ ] Add auto fixing to a rule
```
Comment (bmish) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: I would personally consider `autofix` to be a single word. In our codebase, it's also more common to refer to it as a single word.

Comment (snitin315) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: Makes sense to me 👍

Comment (nzakas) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: Yeah, we generally use “autofix”, so I’d say let’s keep this as-is.

Comment (snitin315) [reply .github/PULL_REQUEST_TEMPLATE.md:22]:
```suggestion
[ ] Add autofix to a rule
```

**Review by nzakas — APPROVED** [review]

**Pull request merged**